instrumentation/opentelemetry-api/opentelemetry-api-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/ContextStorageWrappersInstrumentation.java

```diff
@@ -41,13 +42,16 @@ public void transform(TypeTransformer transformer) {
   @SuppressWarnings("unused")
   public static class AddWrapperAdvice {
 
+    @AssignReturned.ToReturned
     @Advice.OnMethodExit(onThrowable = Throwable.class, suppress = Throwable.class)
-    public static void methodExit(
-        @Advice.Return(readOnly = false)
-            List<Function<? super ContextStorage, ? extends ContextStorage>> wrappers) {
+    public static List<Function<? super ContextStorage, ? extends ContextStorage>> methodExit(
+        @Advice.Return
+            List<Function<? super ContextStorage, ? extends ContextStorage>> originalWrappers) {
+      List<Function<? super ContextStorage, ? extends ContextStorage>> wrappers = originalWrappers;
       wrappers = new ArrayList<>(wrappers);
```

Comment:
could merge these lines and have `wrappers = new ArrayList<>(originalWrappers);`